src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package duke.task;
+
+public class Deadline extends Task{
```

Comment:
Leave a space before the curly bracket, eg. `extends Task {`, for consistency sake.